session.go

```diff
@@ -44,35 +44,35 @@ func Create(username, password, launcherToken, gameToken string) *Session {
 	// Prepare request.
 	req, err := c.NewRequest(http.MethodPost, oauthTokenURL, strings.NewReader(data.Encode()))
 	if err != nil {
-		log.Fatalln(err)
+		return nil, "", err
```

Comment:
These will never be reached since `log.Fatalln()` calls `os.Exit()`. I declared these fatal since the building of the request _should_ never fail, in theory, and if it does, something is very, very wrong with the setup of the request.